client/a8c-for-agencies/components/sidebar/style.scss

```diff
@@ -0,0 +1,69 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+// This file is meant to house style rules that all implementations of the base
+// sidebar will implement by default.
+.a4a-sidebar {
+	.a4a-sidebar__site-selector {
+		background-color: var(--color-sidebar-background);
+	}
+
+	.sidebar-v2__menu-item {
+		svg {
+			color: #6f6f6f;
+		}
+		&.is-active {
+			background: var(--studio-green-0);
+			color: var(--studio-green-50);
+
+			svg {
+				color: var(--studio-green-50);
+			}
+		}
```

Comment:
[NITPICK] Do you think we can keep a constant for A4A-specific colors on this file and update it with the variable once we have the theme?

Reply:
Good point. I made some changes and used some existing variables (--color-sidebar-menu-*), which we can override later in the theme once we have the colors available.